### DISCUSSION TITLE
Why tempo 2.5.x version is not released

### DISCUSSION THREAD

**Discussion by NeerajK23** [grafana/tempo#3972] [General]
It more than 2.5 month and tempo sub version is not even release. There are many changes and commit done to fix security and other bug fix. This is first time there is on sub version is release even after 2.5 months. Atleast tempo 2.5.1 should be release.

**Comment by joe-elliott** [comment]
We do about 4 minor releases of Tempo a year or about 1 every 3 months. 2.6 should be released on schedule in the next few weeks.
